rigex-front-end/src/components/RigSidebar/RigList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { List, ListItemText, MenuItem } from '@material-ui/core';
+
+import './sideBar.scss';
+
+function RigList(props) {
```

Comment:
is there a particular reason why this isn't an arrow function?
If there isn't, please update this to be an arrow function

Reply:
also, you can destructure `props` here to be `({ rigList })`. That way you can just call `rigList` instead of `props.rigList`

you seem to be using rigList twice, but nothing else from props nor are you passing the whole props object to a child, so you don't need the whole object after this point